site/docs/guide/errors.md

```diff
@@ -137,13 +137,33 @@ function errorHandler(err: BotError) {
 }
 ```
 
-In the above example, the `boundaryHandler` handler will be invoked for
+In the above example, the `boundaryHandler` will be invoked for
 
-1. all middleware that is passed to `bot.errorBoundary` after `boundaryHandler` (i.e. `Q`), and
-2. all middleware that is installed on subsequently installed composer instances (i.e. `X`, `Y`, and `Z`).
+1. all middlewares that are passed to `bot.errorBoundary` after `boundaryHandler` (i.e. `Q`), and
+2. all middlewares that are installed on subsequently installed composer instances (i.e. `X`, `Y`, and `Z`).
 
 > Regarding point 2, you may want to skip ahead to [the advanced explanation](/advanced/middleware.md) of middleware to learn how chaining works in grammY.
 
+You can also apply an error boundary to a composer without calling `bot.errorBoundary`:
+
+```ts
+const composer = new Composer();
+
+const protected = composer.errorBoundary(boundaryHandler);
+protected.use(/* B */);
+
+bot.use(composer);
+bot.use(/* C */);
+
+function boundaryHandler(err: BotError, next: NextFunction) {
+  throw new Error("Error in B!");
```

Comment:
This needs more explanation, similar to the comment in the sniper above.

Reply:
I was rather thinking for something like this:
```suggestion
  console.error("Error in Q, X, Y, or Z!", err);
  /*
   * You could call `next` if you want to run
   * the middleware at C in case of an error:
   */
  // await next()
  throw new Error("Error in B!");
```